tests/actions/pastOrders.test.js

```diff
@@ -1,36 +1,76 @@
 import {
   LOAD_PAST_ORDERS_SUCCESS,
-  LOAD_PAST_ORDERS_FAILURE
+  LOAD_PAST_ORDERS_FAILURE,
+  NETWORK_ERROR
 } from '../../app/js/actions/actionTypes';
 import { getPastOrders } from '../../app/js/actions/pastOrders';
 
-window.location = locationMock;
-
-const uuid = '761d8414-6cb4-4456-be4f-232e6f819e06';
+const limit = 10;
```

Comment:
@flacode I would suggest you make use of the mockData file

Reply:
@Betty-Kebenei, could you please help me understand this.

Reply:
@Betty-Kebenei do you mind helping her understand that? 😊